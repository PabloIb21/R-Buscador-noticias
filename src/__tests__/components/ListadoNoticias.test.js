import React from 'react';
import { mount } from 'enzyme';
import ListadoNoticias from '../../components/ListadoNoticias';

describe('<ListadoNoticias />', () => {
  test('mount', () => {
    const noticias = [
      {
        urlToImage: 'https://google.com/image.jpg', 
        url: 'https://google.com', 
        title: 'Hola Mundo',
        description: 'lorem ipsum',
        source: 'google.com'
      }
    ];

    const wrapper = mount(<ListadoNoticias noticias={noticias} />);

    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('h3').text()).toBe('Hola Mundo');
  });
});