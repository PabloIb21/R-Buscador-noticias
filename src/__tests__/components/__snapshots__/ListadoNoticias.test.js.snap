// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListadoNoticias /> mount 1`] = `
<div
  className="row"
>
  <div
    className="col s12 m6 l4"
  >
    <div
      className="card"
    >
      <div
        className="card-image"
      >
        <img
          alt="Hola Mundo"
          src="https://google.com/image.jpg"
        />
        <span
          className="card-title"
        />
      </div>
      <div
        className="card-content"
      >
        <h3>
          Hola Mundo
        </h3>
        <p>
          lorem ipsum
        </p>
      </div>
      <div
        className="card-action"
      >
        <a
          className="waves-effect waves-light btn"
          href="https://google.com"
          rel="noopener noreferrer"
          target="_blank"
        >
          Ver Noticia Completa
        </a>
      </div>
    </div>
  </div>
</div>
`;
